### ISSUE TITLE
editing attribute has unacceptable value for "multivalued" by default

### ISSUE THREAD

**Issue by willow9886** [GluuFederation/oxTrust#1060]
![image](https://user-images.githubusercontent.com/5271048/43608927-2a7e0834-9668-11e8-9728-1dafe801e947.png)

Trying to edit an attribute in oxTrust, `password`, but for some reason the default value for `multivalued` field is `multivalued` which is an unacceptable value for that field. 

This seems to be the case for editing any attribute. They all have default value for `multivalued` to `multivalued` even though thats not an acceptable value..

So basically to edit any attribute, you also have to set an acceptable value for `multivalued`. Shouldn't the field be set to an acceptable value by default??

**Comment by natt-tester** [comment]
@syntrydy, I think we still need improvement in this issue: I've checked two random Attributes and they are not set to their default value as @willow9886 suggested; the "Multivalued" string is just cleared from the first dropdown option:
![multi2](https://user-images.githubusercontent.com/38302725/45134735-67143100-b1df-11e8-8316-67135b53f826.png)

![multi1](https://user-images.githubusercontent.com/38302725/45134726-5a8fd880-b1df-11e8-80f6-20fcca48f632.png)

**Comment by syntrydy** [comment]
@natt-tester that is the expected behavior.

From the UI point of view, you can add an attribute with **multivalued** set to **null**, however Gluu ship with multiples attributes with that field set to **null**. These are attributes added via scripts and installation time, so no mean to enforce the field value**
